lib/usagereporter/teleport/aggregating/reporter.go

```diff
@@ -271,6 +271,11 @@ func (r *Reporter) run(ctx context.Context) {
 
 		return record
 	}
+	userRecordWithOrigin := func(userName string, v1AlphaUserKind prehogv1alpha.UserKind, v1AlphaUserOrigin prehogv1alpha.UserOrigin) *prehogv1.UserActivityRecord {
+		record := userRecord(userName, v1AlphaUserKind)
+		record.UserOrigin = prehogv1.UserOrigin(v1AlphaUserOrigin)
```

Comment:
Casting enum types here and below. I've put a note on the proto definition to keep the enum values in sync. Open for suggestion to implement explicit conversion.